bucket.go

```diff
@@ -20,6 +20,8 @@ type Bucket interface {
 	GetName() string
 	Get(k string, rv interface{}) error
 	GetRaw(k string) ([]byte, error)
+	Gets(k string, rv interface{}, caso *uint64) error
```

Comment:
Gets and GetsRaw are strictly to align with the existing go-couchbase method names and signatures, but on further thought I'm not actually much of a fan of those.

Ideally we'd keep the API simple, and just replace our `Get(key, interface{}) error` with `Get(key, interface{}) (cas, error)`.  Let me see how much work that would be, instead of adding duplicates.